### PR TITLE
Histogram event added

### PR DESCRIPTION
**Pull request by bilgili**

### CODE REVIEW

File: lexis/render/histogram.fbs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2016, Human Brain Project
+//                     Ahmet Bilgili <ahmet.bilgili@epfl.ch>
+
+namespace lexis.render;
+
+table Histogram
```
Comment (hernando) [inline lexis/render/histogram.fbs]: Document better what this type is used for. Just looking at the members it's difficult to guess (what's incarnationId?, why a viewport area?...).
It looks like this histogram is only valid for 8bit dataset, confirm that in the documentation.

File: lexis/render/histogram.fbs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+{
+  data:[ulong:256];
+  viewportArea:float;
+  incarnationId:uint;
+  min:short;
+  max:short;
+}
+
+root_type Histogram;
```
Comment (hernando) [inline lexis/render/histogram.fbs]: Not needed anymore as far as I know.

File: doc/Changelog.md [hunk 3]
```diff
@@ -2,4 +2,6 @@
 
 # git master
 
+* [#xxx](https://github.com/HBPVis/Lexis/pull/xxx):
```
Comment (hernando) [inline doc/Changelog.md]: update

File: lexis/render/histogram.fbs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  bins:[ulong:256]; // The bins for data.
+  minIndex:short; // Index of minimum valued bin
+  maxIndex:short; // Index of maximum valued bin
+
+// Application specific areas ( i.e. Livre )
+  viewportArea:float; // The viewport area bin is computed.
+  id:uint; // The unique identifier of the histogram
+}
+
+root_type Histogram;
```
Comment (dnachbaur) [inline lexis/render/histogram.fbs]: Not needed

File: lexis/render/viewHistogram.fbs [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+table ViewHistogram
+{
+  bins:[ulong:256]; // The bins for data.
+
+// Application specific areas ( i.e. Livre )
+  area:float; // The viewport area, bin is computed.
+  id:uint; // The unique identifier of the histogram
+}
+
+root_type Histogram;
```
Comment (hernando) [inline lexis/render/viewHistogram.fbs]: remove

**Comment by hernando** [comment]
You still haven't updated the PR number in the changelog.

File: doc/Changelog.md (hunk 3 above)
Comment (bilgili) [inline doc/Changelog.md]: done

**Comment by hernando** [comment]
+1

**Pull request merged**